cnf-certification-test/suite_test.go

```diff
@@ -131,7 +117,32 @@ func TestTest(t *testing.T) {
 
 	// Set up logging params for logrus
 	loghelper.SetLogFormat()
-	setLogLevel()
+
+	// Delete execution log (if any) prior to run
+	logFile := "tnf-execution.log"
```

Comment:
if _, err := os.Stat("/path/to/whatever"); err == nil

See:
 https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go